src/components/ChatComponent/ChatComponent.js

```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components";
+import NamebarTop from "./NamebarTop";
+import Chatbar from "./Chatbar";
+import ChatBubble from "./MessageBubble";
+
+const ChatContainer = styled.div`
```

Comment:
I'm not loving that some times the styling consts are above and sometimes below the jsx. Pick one method as a group, fx:
- JS
- JSX
- styling